### PR TITLE
INSTALL: Update with full dependency instructions

### PR DESCRIPTION
**Pull request by davelab6**
Mac OS X only though

### CODE REVIEW

File: INSTALL.txt [hunk 1]
```diff
@@ -1,4 +1,60 @@
...
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" ;
+    brew install python3 ;
+
+## PyQt5
+
+Learn more: https://www.riverbankcomputing.com/software/pyqt
+
+MacOS X: Install with homebrew, or re-install with DBus support,
+
+    brew remove qt5 && brew install qt5 --with-d-bus ;
```
Comment (adrientetar) [inline INSTALL.txt]: brew install pyqt5 will install the dependency automatically.

**Comment by adrientetar** [comment]
Please make an OSX section for that content.

**Comment by adrientetar** [comment]
Does someone want to write instructions for GNU? travis.yml in the repo is a good reference for that. For Windows it's the same except there's an installer for PyQt.

File: INSTALL.txt (hunk 1 above)
Comment (davelab6) [inline INSTALL.txt]: On Oct 19, 2015 6:30 PM, "Adrien Tétar" <notifications@github.com> wrote:
>
> In INSTALL.txt:
>
> >
> > +Learn more: https://docs.python.org/3
> > +
> > +MacOS X: Install with [homebrew](http://brew.sh),
> > +
> > +    sudo xcodebuild -license ;
> > +    ruby -e "$(curl -fsSL
https://raw.githubusercontent.com/Homebrew/install/master/install)" ;
> > +    brew install python3 ;
> > +
> > +## PyQt5
> > +
> > +Learn more: https://www.riverbankcomputing.com/software/pyqt
> > +
> > +MacOS X: Install with homebrew, or re-install with DBus support,
> > +
> > +    brew remove qt5 && brew install qt5 --with-d-bus ;
>
> brew install pyqt5 will install the dependency automatically.

It did not 12 hours ago but there was a PR to brew to fix it

**Comment by pathumego** [comment]
@lpsandaruwan

**Comment by n7s** [comment]
Here's the Ubuntu/Debian version, someone please test this on Fedora:

Installation steps for Ubuntu/Debian
=====================================

(Note that we are using version 3.4 of python).


sudo apt-get install -qq -y python3-pyqt5 python3-flake8

git clone --depth=1 --branch=python3-ufo3 https://github.com/trufont/robofab
git clone --depth=1 --branch=python3-ufo3 https://github.com/trufont/defcon
git clone --depth=1 --branch=python3-ufo3 https://github.com/trufont/ufo2fdk
git clone --depth=1 https://github.com/behdad/fonttools.git

cd defcon
sudo python3.4 setup.py install --record installed-files.txt

cd ../robofab
sudo python3.4 setup.py install --record installed-files.txt

cd ../ufo2fdk
sudo python3.4 setup.py install --record installed-files.txt

cd ../fonttools
sudo python3.4 setup.py install --record installed-files.txt

cd ../trufont
sudo python3.4 setup.py install --record installed-files.txt

To run trufont:
python3.4 -m defconQt


To uninstall cleanly the various modules and trufont
====================================================

The various python modules are installed in
/usr/local/lib/python3.4/dist-packages/

You can delete the subfolders manually or depending on the package, you can also rely on pip or the listing of files you have recorded.

sudo apt-get install python3-pip
sudo pip3 uninstall robofab defcon ufo2fdk ufoLib defconQt

to get rid of all the remaining files installed : (careful with rm)
cat installed-files.txt | xargs sudo rm --verbose -vr

File: INSTALL.txt (hunk 1 above)
Comment (davelab6) [inline INSTALL.txt]: Now merged

**Comment by n7s** [comment]
see https://github.com/trufont/trufont/pull/45

**Comment by davelab6** [comment]
> Here's the Ubuntu/Debian version, someone please test this on Fedora

Thanks @n7s! I've included this text in the above commit.

**Comment by davelab6** [comment]
> see #45

Oops! Sorry, didn't mean to duplicate effort :)

**Comment by adrientetar** [comment]
@davelab6 You can create a new PR for OSX instructions.

**Comment by davelab6** [comment]
Done in https://github.com/trufont/trufont/pull/59

**Pull request closed without merging**